apps/admin_app/assets/css/custom/custom.scss

```diff
@@ -211,12 +238,12 @@ body {
 
       span {
         display: inline-block;
-        padding: 4px 6px;
+        padding: 8px 7px;
         font-weight: 600;
         font-size: 18px;
         line-height: 1.2;
         -webkit-font-smoothing: antialiased;
-        font-family: "Graphik Webfont", -apple-system, BlinkMacSystemFont, "Roboto", "Droid Sans", "Segoe UI", "Helvetica", Arial, sans-serif;
+        font-family: "avia1" !important;
```

Comment:
I think we could apply global font to entire html

Reply:
Ya I have applied for all as well as  specific for this only.